### PR TITLE
fix(dropdown): fix dropdown closing bug

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**Dropdwon**
Shortcut ticket: [sh-216338](https://app.shortcut.com/sci-design-system/story/216338/fix-the-opening-and-closing-functionality-of-the-dropdown-component)

### CODE REVIEW

File: src/core/Dropdown/index.tsx [hunk 1]
```diff
@@ -100,87 +101,133 @@ export default function Dropdown<Multiple extends boolean | undefined = false>({
...
+    </ClickAwayListener>
+  );
+
+  function handleClickAway() {
+    if (open) {
+      setOpen(false);
+
+      if (multiple) {
+        setValue(
+          pendingValue as MUIValue<
```
Comment (mmmmmmaya) [inline src/core/Dropdown/index.tsx:187]: I see this type copy/pasted a few times. Might want to define it somewhere above and give it a shorter name.

File: src/core/Dropdown/index.tsx [hunk 2]
```diff
@@ -208,7 +255,6 @@ export default function Dropdown<Multiple extends boolean | undefined = false>({
     }
 
     if (onClose) onClose();
-    setAnchorEl(null);
```
Comment (mmmmmmaya) [inline src/core/Dropdown/index.tsx:211]: should `setOpen(false)` happen here instead?

**Review by mmmmmmaya — APPROVED** [review]

File: src/core/Dropdown/index.tsx (hunk 1 above)
Comment (masoudmanson) [reply src/core/Dropdown/index.tsx:187]: That makes sense 👍🏻 I'll update the repo.

**Review by mmmmmmaya — APPROVED** [review]

**Review by bethbertozzi — APPROVED** [review]
thanks! looks so much better 🙌

**Pull request merged**